tests/integration/test_format_txts_e2e.py

```
@@ -0,0 +1,266 @@
+import shutil
+import subprocess
+from pathlib import Path
+from unittest.mock import patch, AsyncMock, MagicMock, call
+
+import sys
+# Add the scripts directory to the path
+sys.path.append(str(Path(__file__).parent.parent.parent / "scripts"))
+
+from format_txts import (
+    main,
+    run_imessage_exporter,
```

Ruff (0.8.2)

`format_txts.run_imessage_exporter` imported but unused

Remove unused import

(F401)